packages/atlas/src/views/global/NftPurchaseBottomDrawer/NftPurchaseBottomDrawer.tsx

```diff
@@ -0,0 +1,566 @@
+                  <FlexWrapper>
+                    <Text variant="t100" secondary margin={{ left: 2, right: 1 }}>
+                      {blocksLeft > 0 ? blocksLeft : 0} {blocksLeft === 1 ? 'block' : 'blocks'}
+                    </Text>
+                    <Information
+                      text="Auctions are run and settled on-chain and use blocks of operations rather than clock time."
+                      footer={<Text variant="t100">Auctions closing block: 123115234</Text>}
+                      placement="top"
+                    />
+                  </FlexWrapper>
+                </FlexWrapper>
+              )}
+            </Header>
+            {type !== 'buy_now' && !boughtForFixedPrice ? (
+              <>
+                <CurrentBidWrapper>
+                  {lastBid ? (
+                    <ActiveBidWrapper>
+                      <ActionBarCell>
+                        <Text variant="h300" secondary margin={{ bottom: 2 }}>
+                          Current bid
+                        </Text>
+                        <FlexWrapper>
+                          <CurrentBidAvatar size="bid" />
+                          <BidderName variant="h400">{lastBid?.bidder.handle}</BidderName>
+                        </FlexWrapper>
+                      </ActionBarCell>
+                      <ActionBarCell>
+                        <Text variant="h300" secondary margin={{ bottom: 2 }}>
+                          Bid amount
+                        </Text>
+                        <FlexWrapper>
+                          <CurrentBidJoyToken size={24} variant="silver" />
+                          <Text variant="h400">{lastBid?.amount}</Text>
+                        </FlexWrapper>
+                      </ActionBarCell>
+                    </ActiveBidWrapper>
+                  ) : (
+                    <ActiveBidWrapper>
+                      <ActionBarCell>
+                        <Text variant="h300" secondary margin={{ bottom: 2 }}>
+                          Current bid
+                        </Text>
+                        <Text variant="h400">Nobody has bid yet</Text>
+                      </ActionBarCell>
+                    </ActiveBidWrapper>
+                  )}
+                </CurrentBidWrapper>
+                {!auctionEnded && (
+                  <MinimumBidWrapper>
+                    <MinimumBid>
+                      <Text variant="h300" secondary>
+                        Minimum bid:
+                      </Text>
+                      <JoyTokenIcon variant="silver" size={24} /> <Text variant="h400">{calculatedMinimumBid}</Text>
+                    </MinimumBid>
+                    <div>
+                      <Text variant="t100" secondary>
+                        Buy now: {auctionBuyNowPrice} tJOY
+                      </Text>
+                    </div>
+                  </MinimumBidWrapper>
+                )}
+                <TextField
+                  {...register('bid', {
```

Comment:
Validation here happens instantly. That is, I start typing and I immediately see an error. I should see an error only if I blur the field or try to submit, depending on the form mode. Previously it would validate only at submit

Reply:
Okay, I changed that, but there is a problem with this - in the design, we have a disabled button when the form is not valid.
Let's think about the scenario: 
1. minimum bid is 300 
2. user is typing 200
3. user sees the disabled button and has no idea why. I think this is a little bit misleading. This is what he sees: 👇🏻 
![image](https://user-images.githubusercontent.com/51168865/158997229-b465476f-b4b1-4fb8-a798-e5a517152207.png)
4. Error will show only if he clicks anywhere on the page